### DISCUSSION TITLE
Nuxt 3 support roadmap

### DISCUSSION THREAD

**Discussion by StefanVDWeide** [nuxt-community/sitemap-module#265] [Q&A]
Now that the Nuxt 3 stable version is approaching fast, RC.8 at the time of writing, I was wondering if there is a roadmap available for Nuxt 3 support.

This is a great module that I'm happily using in my Nuxt 2 projects, hopefully I can keep using the in the future! Thanks for the hard work! :)

**Comment by thisismydesign** [comment]
I made a workaround for Nuxt 2 Bridge, might work for Nuxt 3 as well: https://github.com/nuxt-community/sitemap-module/issues/281

**Comment by surgiie** [comment]
This package is starting to look abandoned to me. I dont see any activity and last commit to the dev branch was 2021. :/
